src/commands/daily.ts

```
@@ -46,6 +46,7 @@ export const dailyCommand = define({
 			groupByProject: ctx.values.instances,
 			project: ctx.values.project,
 			timezone: ctx.values.timezone,
+			locale: ctx.values.locale,
```

ESLint

[error] Unsafe assignment of an `any` value.

(ts/no-unsafe-assignment)